### PR TITLE
Added price and quantity for the plenigo checkout button

### PR DESCRIPTION
**Pull request by retorres87**

### CODE REVIEW

File: plenigo_js/tinymce-plenigo-plugin.js [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
+}
 tinymce.PluginManager.add('plenigo', function (editor, url) {
     // Add a button that opens a setup window and configures the checkout button
     editor.addButton('plenigo', {
         tooltip: 'Product Checkout Button',
         text: '',
         icon: 'pl-checkout',
         onclick: function () {
+            var quantityValues = [];
+            for(var i = 1; i <= 50; i++) {
```
Comment (robertholfeld) [inline plenigo_js/tinymce-plenigo-plugin.js]: Formatting

File: plenigo_js/tinymce-plenigo-plugin.js [hunk 2]
```diff
@@ -33,8 +43,19 @@ tinymce.PluginManager.add('plenigo', function (editor, url) {
...
                     var affiliate_text = '';
                     var class_text = '';
+                    var price_text = '';
+                    var isInvalid = false;
+                    var price = e.data.price.trim();
+                    var quantity = e.data.quantity.trim();
                     if (e.data.prodId.trim() === '' || e.data.prodId.length < 5) {
                         editor.windowManager.alert('Invalid Product ID!');
+                        isInvalid = true;
+                    }else if (isInvalidNumber(price)) {
```
Comment (robertholfeld) [inline plenigo_js/tinymce-plenigo-plugin.js]: Formatting

File: plenigo_js/tinymce-plenigo-plugin.js [hunk 3]
```diff
@@ -33,8 +43,19 @@ tinymce.PluginManager.add('plenigo', function (editor, url) {
...
+                        editor.windowManager.alert('Invalid Price!');
+                        isInvalid = true;
+                    } else if(isInvalidNumber(quantity)) {
```
Comment (robertholfeld) [inline plenigo_js/tinymce-plenigo-plugin.js]: Formatting

File: plenigo_js/tinymce-plenigo-plugin.js [hunk 4]
```diff
@@ -33,8 +43,19 @@ tinymce.PluginManager.add('plenigo', function (editor, url) {
                     var target_text = '';
                     var affiliate_text = '';
                     var class_text = '';
+                    var price_text = '';
+                    var isInvalid = false;
+                    var price = e.data.price.trim();
```
Comment (robertholfeld) [inline plenigo_js/tinymce-plenigo-plugin.js]: null check of price?

File: plenigo_js/tinymce-plenigo-plugin.js [hunk 5]
```diff
@@ -33,8 +43,19 @@ tinymce.PluginManager.add('plenigo', function (editor, url) {
...
+                    var quantity = e.data.quantity.trim();
```
Comment (robertholfeld) [inline plenigo_js/tinymce-plenigo-plugin.js]: null check of quantity

File: plenigo_plugin/PlenigoShortcodeManager.php [hunk 6]
```diff
@@ -158,16 +158,20 @@ public function plenigo_handle_shortcode($atts, $content = null, $tag = null) {
...
+	    error_log("Attributes sent" . print_r($atts, true));
+	    error_log("Attributes filled" . print_r($a, true));
         $btnTitle = $a['title'];
         $cssClass = $a['class'];
         $prodId = $a['prod_id'];
         $regCheck = $a['register'];
         $sourceURL = $a['source'];
         $targetURL = $a['target'];
         $affiliate = $a['affiliate'];
+	    $price = $a['price'];
```
Comment (robertholfeld) [inline plenigo_plugin/PlenigoShortcodeManager.php]: Formatting

File: plenigo_js/tinymce-plenigo-plugin.js (hunk 5 above)
Comment (retorres87) [reply plenigo_js/tinymce-plenigo-plugin.js]: the data by default is an empty string when it is not filled, this is why i don't do null checks

File: plenigo_js/tinymce-plenigo-plugin.js (hunk 4 above)
Comment (retorres87) [reply plenigo_js/tinymce-plenigo-plugin.js]: the data by default is an empty string when it is not filled, this is why i don't do null checks

**Pull request merged**